bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -1096,6 +1098,36 @@ def test_put_user_operator_operator(self):
         assert operator.updated_by == self.user
         assert operator.updated_at is not None
 
+    def test_put_user_operator_operator_unauthorized(self):
+        operator = operator_baker()
+        operator.created_by = self.user
+        operator.save(update_fields=["created_by"])
+        mock_payload = {
+            "legal_name": "Unauthorized",
+            "trade_name": "Unauthorized",
+            "cra_business_number": 678123654,
+            "bc_corporate_registry_number": "jkl1234321",
+            "business_structure": BusinessStructure.objects.first().pk,
+            "physical_street_address": "add",
+            "physical_municipality": "add",
+            "physical_province": "BC",
+            "physical_postal_code": "H0H0H0",
+            "mailing_address_same_as_physical": True,
+            "operator_has_parent_operators": False,
+        }
+        user_operator = baker.make(
+            UserOperator, user=self.user, operator=operator, role=UserOperator.Roles.REPORTER, created_by=self.user
```

Comment:
A `pending` role might be a better test case? I don't think we'll ever have anyone with a reporter role for MVP.

Reply:
Good call, I'll test for both.